src/workerd/api/actor-state.c++

```diff
@@ -297,15 +306,17 @@ jsg::Promise<kj::Maybe<double>> DurableObjectStorageOperations::getAlarm(
   }).orDefault(GetOptions{}));
   auto result = getCache(OP_GET_ALARM).getAlarm(options);
 
-  return transformCacheResult(
-      js, kj::mv(result), options, [](jsg::Lock&, kj::Maybe<kj::Date> date) {
+  return transformCacheResult(js, kj::mv(result), options,
+      [userSpan = kj::mv(userSpan)](jsg::Lock&, kj::Maybe<kj::Date> date) {
```

Comment:
This attachment will work in this case but mostly by accident.
Can we use the whenResolved method here as well for consistency please?